### PR TITLE
ZK-4091: Collapsing a Groupbox with invalidate causes a JavaScript error

### PR DESCRIPTION
**Pull request by scribetw**

### CODE REVIEW

File: zul/src/archive/web/js/zul/wgt/Button.js [hunk 1]
```diff
@@ -228,7 +228,7 @@ zul.wgt.Button = zk.$extends(zul.LabelImageWidget, {
 		// Bug ZK-1295 and ZK-2935: Disabled buttons cannot regain focus by re-enabling and then setting focus
 		var wgt = this,
 			btn = this.$n();
-		if (btn.disabled) {
+		if (btn && btn.disabled) {
```
Comment (scribetw) [inline zul/src/archive/web/js/zul/wgt/Button.js]: zk(btn).focus

**Comment by DevChu** [comment]
@zkadmin rebase

**Comment by zkadmin** [comment]
I just pushed up the changes, enjoy!

**Comment by charlesqqq** [comment]
@zkadmin rebase

**Comment by zkadmin** [comment]
I just pushed up the changes, enjoy!

**Pull request merged**